vertical-pod-autoscaler/pkg/recommender/main.go

```diff
@@ -116,6 +119,9 @@ const (
 )
 
 func main() {
+	flag.Var(&maxAllowedCPU, "container-recommendation-max-allowed-cpu", "Maximum amount of CPU that will be recommended for a container. VerticalPodAutoscaler-level maximum allowed takes precedence over the global maximum allowed.")
```

Comment:
Let's move these into an `init()` function in this file?

https://go.dev/doc/effective_go#init

It is guaranteed to run before `main()`.